projects/storefrontlib/src/cms-components/storefinder/components/store-finder-store/store-finder-store.component.html

```diff
@@ -9,7 +9,7 @@
     <div class="col-md-4 col-sm-6 col-lg-2">
       <button class="btn btn-block btn-action" (click)="goBack()">
         <cx-icon [type]="iconTypes.CARET_LEFT"></cx-icon>
-        {{ 'storeFinder.backToList' | cxTranslate }}
+        {{ 'storeFinder.clickBack' | cxTranslate }}
```

Comment:
Looks unrelated to this PR. Should be reverted.

Reply:
weird. I've reverted.